velox/exec/tests/OutputBufferManagerTest.cpp

```diff
@@ -971,6 +977,60 @@ TEST_P(AllOutputBufferManagerTest, outputBufferUtilization) {
   verifyOutputBuffer(task, OutputBufferStatus::kFinished);
 }
 
+TEST_P(AllOutputBufferManagerTest, outputBufferStats) {
+  const vector_size_t size = 100;
+  const std::string taskId = std::to_string(rand());
```

Comment:
@HolyLow Meta internal static linting is suggesting following
```
using C rand() to generate random number causes lock contention, please consider folly::Random::rand32().
```
Lets remove rand() here and use [folly](folly::Random::rand32() instead.

Reply:
done.